docs/DeveloperGuide.md

```diff
@@ -237,70 +237,403 @@ _{Explain here how the data archiving feature will be implemented}_
 **Target user profile**:
 
 * has a need to manage a significant number of contacts
+* has a need to manage a meeting schedule
+* has a need to oversee the development of the product
 * prefer desktop apps over other types
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**: Used in a corporate setting, our application integrates both a meeting schedule and a contact list for the user to 
+manage their product’s development more comprehensively and conveniently.
 
 
 ### User stories
 
-Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
+Priorities: 
+* `* * *` - High (must have)
+* `* *` - Medium (nice to have)
+* `*` - Low (unlikely to have) 
+
+| Priority | As a …​                               | I want to …​                     | So that I can…​                                                         |
+| -------- | ---------------------------------------- | --------------------------------------------------------------------------- | ---------------------------------- |
+| **Deliverables** |
+| `* * *`  | Product Manager                          | add deliverables                                                            | keep track of them                 |
+| `* * *`  | Product Manager                          | change the description of the deliverables                                  |                                    |
+| `* * *`  | Product Manager                          | mark these deliverables as completed                                        | refer back to them when needed     |
+| `* * *`  | Product Manager                          | delete deliverables that are no longer relevant                             | focus on current deliverables      |
+| `* *`    | Product Manager                          | tag deliverables as delayed and change the deadlines                        |                                    |
+| `* *`    | Product Manager                          | change the person-in-charge of deliverables                                 | locate a person easily             |
+| `* *`    | Product Manager                          | undo my action / be prompted before any action                              | Minimise mistakes made             |
+| `* *`    | Product Manager                          | view the overall completion percentage of the project                       | know whether we are on track (completed deliverables / total deliverables) |
```

Comment:
Should we take out "(completed deliverables / total deliverables)" since it may lead to confusion? It may be misunderstood as completed deliverables _or_ total deliverables instead of _over_. Alternatively to explicitly state "completed deliverables out of total deliverables".